frontend/src/components/admin/ManageUsers.tsx

```diff
@@ -1,27 +1,38 @@
 import React, { useState } from "react";
 import { useQuery } from "@apollo/client";
 import { Heading } from "@chakra-ui/react";
-import UsersTable from "./UsersTable";
+import UsersTable, { alphabeticalCompare } from "./UsersTable";
 import { buildUsersQuery, User } from "../../APIClients/queries/UserQueries";
+import deepCopy from "../../utils/DeepCopyUtils";
 
-const ManageReviewers = () => {
+export type ManageUsersProps = {
+  isTranslators: boolean;
+};
+
+const ManageUsers = ({ isTranslators }: ManageUsersProps) => {
   const query = buildUsersQuery();
   const [users, setUsers] = useState<User[]>([]);
   useQuery(query.string, {
     fetchPolicy: "cache-and-network",
     onCompleted: (data) => {
-      setUsers(data[query.fieldName]);
+      const newUsers = deepCopy(data[query.fieldName]);
```

Comment:
just curious, why do you use `deepCopy` here and in `UsersTable`?

Reply:
oh good point! not sure where my head was. Spread operator copy works perfectly fine in both places, updated the code